diesel_tests/tests/debug/mod.rs

```diff
@@ -1,15 +1,25 @@
+use schema::TestBackend;
 use diesel::*;
 
 #[test]
 fn test_debug_count_output() {
     use schema::users::dsl::*;
-    let sql = debug_sql!(users.count());
-    assert_eq!(sql, "SELECT COUNT(*) FROM `users`");
+    let sql = debug_sql::<TestBackend, _>(&users.count());
```

Comment:
Perhaps instead of a `debug_sql` function, we should have a `.debug()` instance method which just returns a struct that implements `std::fmt::Debug`